extensions/todolist@bsaleil.org/Makefile.am

```diff
@@ -0,0 +1,14 @@
+todoschemadir = $(extensiondir)/schemas
```

Comment:
What I meant is to replace [this code](https://github.com/kinvolk/coding-shell-extensions/blob/lili/clean_deb/extensions/todolist%40bsaleil.org/utils.js#L13-L19) with [this code](https://github.com/kinvolk/coding-shell-extensions/blob/lili/clean_deb/extensions/time_tracker_jsnjack%40gmail.com/convenience.js#L77-L84).